infra/cifuzz/build_fuzzers.py

```diff
@@ -110,6 +110,13 @@ def build_fuzzers(self):
       self.handle_msan_postbuild(docker_container)
     return True
 
+  def upload_build(self):
+    """Upload build."""
+    if self.config.builds_storage:
+      self.clusterfuzz_deployment.upload_latest_build()
```

Comment:
Should we return this value?

Reply:
Maybe not. I don't think we're consistent in clusterfuzz_deployment about returning things.